src/types/stardust/mod.rs

```diff
@@ -5,3 +5,6 @@
 pub mod block;
 /// Module containing the milestone model.
 pub mod milestone;
+
+/// Module containing utilities for stardust types.
+pub mod util;
```

Comment:
These utility functions should live with the integration tests in the `common` module.